src/libraries/System.IO.FileSystem/tests/Enumeration/AttributeTests.cs

```diff
@@ -113,18 +113,42 @@ public void DirectoryAttributesAreExpected()
         }
 
         [Fact]
-        public void IsHiddenAttribute()
+        [PlatformSpecific(TestPlatforms.Windows | TestPlatforms.OSX)]
+        public void IsHiddenAttribute_Windows_OSX()
         {
+            // Put a period in front to make it hidden on Unix
+            IsHiddenAttributeInternal(useDotPrefix: false, useHiddenFlag: true);
```

Comment:
Can you also add an OSX test that uses the dot prefix?

Reply:
The test `IsHiddenAttribute_Unix` in line 127 runs in OSX too (`TestPlatforms.AnyUnix` is `true` in OSX).